src/components/NewTask.module.scss

```diff
@@ -0,0 +1,18 @@
+.footer--new-task {
+  margin: 0px;
+  display: flex;
+  align-items: center;
+  line-height: 1.1;
+  font-weight: 500;
+  gap: 0px;
```

Comment:
I think this can be removed

Reply:
"align-items: center" is needed but the rest might be removed.